source/docs/tag-plugins.md

```diff
@@ -278,6 +280,19 @@ For instance:
 
 {% post_link 2018-10-19-hexo-3-8-released 'Link to a post' %}
 
+**Escape title.**
+
+```
+{% post_link hexo-4-0-released 'How to use <b> tag in title' %}
+```
+{% post_link hexo-3-9-released 'How to use <b> tag in title' %}
+
+**Do not escape title.**
+
+```
+{% post_link hexo-4-0-released '<b>bold</b> custom title' false %}
+```
+{% post_link hexo-3-9-released '<b>bold</b> custom title' false %}
```

Comment:
Currently it's rendered as `bold custom title false`. This is a [limitation](https://github.com/hexojs/hexo/issues/1277#issuecomment-165827291) of the way Swig parse the parameter.

As workaround I think is,
```
{% post_link hexo-3-9-released '"<b>bold</b> custom title"' '"false"' %}
```

Reply:
ok, I will check it out.

Reply:
Can you confirm whether dependecy of hexo was updated.

### with hexojs/hexo(latest)
<img width="757" alt="2019101003" src="https://user-images.githubusercontent.com/51783821/66538595-b6056600-eb57-11e9-851d-589ca224ab53.png">

### with hexo@3.9.0
<img width="799" alt="2019101002" src="https://user-images.githubusercontent.com/51783821/66538605-c1589180-eb57-11e9-8aec-769db455848e.png">

Reply:
probably is the npm cache as initially suspected.

@hexojs/core 
anyone with access to netlify console, can you try [clearing the cache](https://www.netlify.com/docs/build-gotchas/)?

Reply:
In the meantime, can you try

https://github.com/hexojs/site/blob/da3355ac5fa32982ce221df3d22746a18ae68a5e/netlify.toml#L2

```
command = "npm install hexo@hexojs/hexo && npm run build"
```

Reply:
@curbengh I updated `command` in site/netlify.toml

Reply:
Rendered result is correct on Netlify now.
https://5d9ef7939444aa0007ae1ad2--hexo-site.netlify.com/docs/tag-plugins#Include-Posts

Reply:
@dailyrandomphoto It is not recommends to use `npm install hexo@hexojs/hexo` in netlify build command. Since it will bypass the cache and slow down the build. But since we have no access to Netlify dashboard to purge the cache, I think it will be fine.

Reply:
To clarify, I meant just a temporary trial because I want to make sure the rendered page is ok.

The preview does look ok now, I think you can revert the `npm install`.

Reply:
I found this [reply](https://stackoverflow.com/a/50820392) (seemingly from Netlify):

>  We intend to re-run npm install or yarn install if the checksum on /package.json or /yarn.lock changes

If the page is not rendered correctly after merging this, then we need to

``` json
  "hexo": "hexojs/hexo#commit-hash"
```

to trigger `npm install`, then revert to the usual `"hexojs/hexo"`.

Reply:
After update to `"hexo": "hexojs/hexo#commit-hash"`, the page is rendered correctly. But when revert to the usual `"hexojs/hexo"`, it still use the old cached version. 
So I have left it as `"hexo": "hexojs/hexo#commit-hash"`.

Reply:
@SukkaW 
I mean in the package.json (after merging). we should restore netlify.toml to the original.

@dailyrandomphoto
That's worrying. I figured reverting back will also trigger `npm install`, somehow it's not.

To all,

It seems netlify's `npm install` is quite erratic,

what about in netlify.toml,

```
command = "npm install && npm run build"
```

temporarily, hopefully the cache gets updated then `npm install` can be removed.

Just the netlify config, keep using `"hexo": "hexojs/hexo"` in package.json.